coordinator/ethereum/gas-pricing/static-cap/src/test/kotlin/net/consensys/linea/ethereum/gaspricing/staticcap/ExtraDataV1PricerServiceTest.kt

```diff
@@ -49,9 +49,9 @@ class ExtraDataV1PricerServiceTest {
   )
 
   @Test
-  @Timeout(2, timeUnit = TimeUnit.SECONDS)
+  @Timeout(value = 5, timeUnit = TimeUnit.SECONDS)
```

Comment:
I don't think we need to increase the timeout from 2 to 5 sec, if not necessary, we don't want to increase the test timeout, keep it as 2 sec should be fine